Tests/DependencyInjection/AbstractDoctrineExtensionTest.php

```diff
@@ -399,6 +399,22 @@ public function testEntityManagerMemcacheMetadataCacheDriverConfiguration()
         ));
     }
 
+    public function testEntityManagerRedisMetadataCacheDriverConfiguration()
```

Comment:
method name changed. No idea why the test that fail on PHP 5.5. I keep investigate